Documentation/bpf/architecture.rst

```diff
@@ -807,5 +814,8 @@ The Linux kernel provides few sysctls that are BPF related and covered in this s
   +-------+-------------------------------------------------------------------+
   | 0     | Unprivileged use of bpf syscall enabled (kernel's default value)  |
   +-------+-------------------------------------------------------------------+
-  | 1     | Unprivileged use of bpf syscall disabled                          |
+  | 1     | Unprivileged use of bpf syscall disabled (until reboot)           |
+  +-------+-------------------------------------------------------------------+
+  | 2     | Unprivileged use of bpf syscall disabled                          |
+  |       | (default if ``BPF_UNPRIV_DEFAULT_OFF`` is enabled in kernel config) |
```

Comment:
See how this change breaks the right-side delimiter line for the table. Can you please fix this by updating the other lines of the table, please? I suspect at least some of the CI workflow fail because of that.

Could you please also rebase on top of the current master branch?

Reply:
@qmonnet 
Done